pydantic/schema.py

```diff
@@ -788,6 +788,7 @@ def field_singleton_schema(  # noqa: C901 (ignore complexity)
     f_schema: Dict[str, Any] = {}
     if field.field_info is not None and field.field_info.const:
         f_schema['const'] = field.default
+
```

Comment:
Did you mean to add this whitespace change?

Reply:
Yes